rotkehlchen/chain/ethereum/manager.py

```diff
@@ -526,79 +525,101 @@ def _get_code(self, web3: Optional[Web3], account: ChecksumEthAddress) -> str:
 
         return hex_or_bytes_to_str(web3.eth.getCode(account))
 
+    @overload
     def ens_lookup(
             self,
             name: str,
+            blockchain: Literal[SupportedBlockchain.ETHEREUM] = SupportedBlockchain.ETHEREUM,
             call_order: Optional[Sequence[NodeName]] = None,
+    ) -> Optional[ChecksumEthAddress]:
+        ...
+
+    @overload
+    def ens_lookup(
+            self,
+            name: str,
+            blockchain: Literal[
```

Comment:
Due to this overload, `blockchain` and `call_order` params have swapped positions.